src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+    static Task[] tasks = new Task[100];
```

Comment:
Perhaps look into using an ArrayList might be better? It allows the array to be dynamically sized.